tests/tests_ovirt_elasticsearch.yml

```diff
@@ -415,6 +425,17 @@
         - "retryfailures=.off."
       changed_when: false
 
+    - name: Check server param in "{{ __test_es_conf }}"
+      command: >-
+        /bin/grep "server=\['logging-es0', 'logging-es1'\]" {{ __test_es_conf }}
+      changed_when: false
+
+    - name: Check server param in "{{ __test_es_ops_conf }}"
+      command: >-
+        /bin/grep "server=\['logging-es-ops0', 'logging-es-ops1'\]"
+                  {{ __test_es_ops_conf }}
```

Comment:
Why is this line intended? I believe it must be on the same level as the previous line. Otherwise, lgtm.

Reply:
I think you mean "indented"

Reply:
Right. I wanted to ask if the indentation is intended :)

Reply:
Yes, intended. I just thought it's easier to read. :) The line is a one-line command line... The two lines will be merged into:
```
/bin/grep "server=\['logging-es-ops0', 'logging-es-ops1'\]"          {{ __test_es_ops_conf }}
```
If it looks odd, I will eliminate the white spaces.